docker-compose.yml

```diff
@@ -107,12 +113,13 @@ services:
     container_name: prepare-a-case-cypress
     environment:
       - CYPRESS_FAIL_FAST_ENABLED=true
+      - CYPRESS_CASES_TOTAL_DAYS=${CASES_TOTAL_DAYS:-28}
       - DISPLAY=${DISPLAY}
     working_dir: /e2e
     command: ["open", "-C", "cypress/cypress.config.js"]
     volumes:
       #mac
-      #- /tmp/.X11-unix:/tmp/.X11-unix
+      - /tmp/.X11-unix:/tmp/.X11-unix
```

Comment:
Pretty sure the cypress env isn't mac and this will break it?, perhaps.

Reply:
this works for me in terms of exposing the cypress ui (and pipeline seems to be working fine?). I think we all have macs as devs by standard anyway

Reply:
FYI it also seems to work without it, so I've removed it